packages/shared-ux/file/file_picker/impl/src/file_picker.tsx

```diff
@@ -55,6 +55,10 @@ export interface Props<Kind extends string = string> {
    * When a user has successfully uploaded some files this callback will be called
    */
   onUpload?: (done: DoneNotification[]) => void;
+  /**
+   * `meta` value to be used for file uploads
+   */
+  meta?: FileJSON['meta'];
```

Comment:
Should we call this more clearly on this level? maybe `uploadMeta` or `uploadFileMeta`

Reply:
Yeah good point, initially I thought to just keep it the same as with `FileUpload` and rely on the doc comment for clarity (the ["go" way](https://go.dev/doc/effective_go#package-names) last paragraph of that section).

But seeing it now I agree can be clearer without making it too long. I'll go with `uploadMeta` for now!